steering-committee/prioritization-criteria.md

```diff
@@ -0,0 +1,64 @@
+# OWD prioritization criteria
+The [OWD charter](https://github.com/openwebdocs/project/blob/main/charter.md) says _"prioritize based on the needs of the global community of web developers and designers"_. This document describes criteria for open and sponsor/vendor-neutral prioritization and will further guide OWD’s decision-making.
+
+## Criteria
+In no particular order. When deciding about what to prioritize and which opportunity to work on, OWD will take into account:
+
+### Effort
+* A higher cost ranks negatively
+* How much time needs to be spent?
+* How many pages to write?
+* How complex is the problem?
+* If new pages, how much maintenance will the pages require over time (e.g. how much in flux is the relevant technology space)?
+
+### Dependencies
+* More dependencies rank negatively
+* Does it need a change in the platform?
+* Does it need involvement from external people? (e.g. spec authors, spec tooling, etc.)
+
+### Community enablement
+* More enablement ranks positively
+* Is the work shareable?
+* Does the opportunity allow more contributors to join the project?
+* Does the opportunity allow contributors to be more effective?
+
+### Momentum
+* More momentum ranks positively
+* Are there enough implementations for a technology (and/or implementation interest)?
+* Is there a lot of community interest?
+* Has the specification stabilized? Is there no longer a risk that APIs will be renamed, redesigned etc.?
+
+### Enabling learners
+* Higher enablement ranks positively
+* Is the opportunity about concepts that need to be learned in order to understand the Web better? (as opposed to just yet another API reference documentation or such things).
+* Does the opportunity contribute an educational pathway for someone to understand a module of the mechanics of the Web?
```

Comment:
"module" is a bit of a funny word to use here. CSS uses "module", sure, but is this common terminology across other tech?

I'd prefer this to be phrased a bit more generically, e.g. "Does the opportunity contribute an educational pathway for someone to understand a key web development-related topic" ?